keps/sig-network/1880-multiple-service-cidrs/README.md

```diff
@@ -403,20 +403,6 @@ until it verifies that the consumer associated to that IP has been deleted too.
 The IPAddress will be deleted and an event generated if the controller determines that the IPAddress
 is orphan [(see Allocator section)](#allocator)
 
-- IPAddress referencing recreated Object (different UID)
```

Comment:
What happened to this use case?

Reply:
it was dropped because it didn't have an use case https://github.com/kubernetes/kubernetes/pull/121106